CHANGELOG.md

```diff
@@ -5,7 +5,8 @@ Please see [CONTRIBUTING.md](https://github.com/cucumber/cucumber/blob/master/CO
 ### Changed
  * Renamed History.md to CHANGELOG.md, added contributing note, and this line in accordance with [cucumber/cucumber #251](https://github.com/cucumber/cucumber/issues/251) ([#345](https://github.com/cucumber/cucumber-rails/pull/345) [jaysonesmith](https://github.com/jaysonesmith))
  * Update .travis.yml with ruby versions ([#341](https://github.com/cucumber/cucumber-rails/pull/341) Jun Aruga)
-
+ * Removed support for Ruby <= 2.1, to keep in line with the [core Cucumber-ruby repository](https://github.com/cucumber/cucumber-ruby/blob/master/CHANGELOG.md#302-2017-11-11).
```

Comment:
Tiny detail: `cucumber-ruby` is the name of the repository. We can refer to that casing of the name in the text.

Reply:
Thanks @olleolleolle, I'll make that change now and merge the request. I've also noticed that I've set the minimum cucumber version to 3.0.0 where it should really be 3.0.2 (given that is when Cucumber-ruby officially ceases support for Ruby <= 2.1) - I'll fix this, too.